holoviews/tests/util/test_transform.py

```diff
@@ -520,3 +525,56 @@ def test_pickle(self):
         expr = (((dim('float')-2)*3)**2)
         expr2 = pickle.loads(pickle.dumps(expr))
         self.assertEqual(expr, expr2)
+
+
+@pytest.fixture    
+def df_fixture():
+    """Prepare dataframe fixture.
+
+    Returns:
+        pd.DataFrame: Output data with 
+    """
+    
+    data_text = (
+        """
+        idx| name|   a|    b
+        101|   P1|  7.|   3.
+        102|   P2|  3.|   4.
+        103|   P3|  .5|   3.
+        104|   P4|  2.|   2.
+        105|   P5|  1.|   2.
+        106|   P6|  1.|   1.
+        """)
+
+    # read in data using non-zero based index
+    return pd.read_csv(StringIO(data_text), sep='|', skipinitialspace=True, index_col=0)
```

Comment:
I think you have overengineered this. You could get a similar result by having `pd.DataFrame({"a": [7, 3, 0.5, 2, 1, 1], "b": [3, 4, 3, 2, 2, 1]}, index=range(101, 107))`.

Reply:
I could modify this if you prefer. I just started to appreciate to see tabular input in test where the arrangment matters.

Reply:
I would very much prefer it. It takes focus away from the actual test code.

Reply:
Then I sure will remove it.